src/views/network/view_session.cpp

```diff
@@ -107,9 +128,132 @@ namespace big
 
 		components::sub_title("All Players");
 		ImGui::Checkbox("Off The Radar", &g->session.off_radar_all);
+		ImGui::SameLine();
 		ImGui::Checkbox("Never Wanted", &g->session.never_wanted_all);
+		ImGui::SameLine();
 		ImGui::Checkbox("Semi Godmode", &g->session.semi_godmode_all);
 
+		ImGui::Checkbox("Explosion Karma", &g->session.explosion_karma);
+		ImGui::SameLine();
+		ImGui::Checkbox("Damage Karma", &g->session.damage_karma);
+
+		static int global_wanted_level = 0;
+
+		if (ImGui::SliderInt("Wanted Level", &global_wanted_level, 0, 5))
```

Comment:
Why is `g->session.wanted_level_all` in globals but `global_wanted_level` isn't?

Reply:
That was my assumption as well, the global doesn't need to be controlled outside of this file so it can remain as a static variable.